core/ui/TagTemplate.tid

```diff
@@ -27,4 +33,4 @@ color:$(foregroundColor)$;
 </span>
 \end
 
-<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>
```

Comment:
You are right. I didn't intend on adding that. Github added it and I failed to realize it until you brought it to my attention. I'll have to figure out how to modify my pull request. Thank you for letting me know.


Happy Connecting. Sent from my Sprint Samsung Galaxy S® 5

-------- Original message --------
From: nameanyone <notifications@github.com> 
Date: 5/29/16  9:27 PM  (GMT-07:00) 
To: Jermolene/TiddlyWiki5 <TiddlyWiki5@noreply.github.com> 
Cc: Andrew J Harrison <andrew.j.harrison84@gmail.com>, Mention <mention@noreply.github.com> 
Subject: Re: [Jermolene/TiddlyWiki5] Update TagTemplate.tid (#2469) 

In core/ui/TagTemplate.tid:
> @@ -27,4 +33,4 @@ color:$(foregroundColor)$;
>  </span>
>  \end
>  
> -<$macrocall $name="tag-body" colour={{!!color}} palette={{$:/palette}}/>

@Infurnoape, Some templates do not have a newline at the end of the last line for a reason. This may or may not be one of them. The effect, when it happens, is extra space within the element.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.